packages/react-native/template/metro.config.js

```diff
@@ -1,11 +1,32 @@
-const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+const {getDefaultConfig} = require('@react-native/metro-config');
 
 /**
  * Metro configuration
  * https://facebook.github.io/metro/docs/configuration
  *
  * @type {import('metro-config').MetroConfig}
  */
-const config = {};
+const config = getDefaultConfig(__dirname);
```

Comment:
Clarifying my suggestion from the other comment ⬇️. Ideally, we want end users to understand they can add to the `config` object to supply their own custom config.

```suggestion
const defaultConfig = getDefaultConfig(__dirname);

const config = {
  resolver: {
    // When enabled, the optional code below will allow Metro to resolve
    // and bundle source files with TV-specific extensions
    // (e.g., *.ios.tv.tsx, *.android.tv.tsx, *.tv.tsx)
    //
    // Metro will still resolve source files with standard extensions
    // as usual if TV-specific files are not found for a module.
    //
    // This code is not enabled by default, since it will impact bundling performance,
    // but is available for developers who need this capability.
    // [].concat(
    //   process.env.BUILDING_FOR_TV
    //     ? defaultConfig.resolver.sourceExts.map((e) => `tv.${e}`)
    //     : [],
    //   defaultConfig.resolver.sourceExts,
    // ),
  },
};

module.exports = mergeConfig(defaultConfig, config);
```